_sass/components/_va-playbook.scss

```diff
@@ -0,0 +1,480 @@
+// Adds external icon to all links that begin
+// with http (including https) 
+[href^=http] {
```

Comment:
To `https://vets.gov`, probably yes as a just-in-case measure. I believe all of our links to vets.gov pages are relative, but I'm sure someone will get that wrong in more than one place. 

As for va.gov: I believe most of our external links point to va.gov content. It is another domain that we don't exactly manage. I think the external icons there are fine.